Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -201,7 +201,8 @@ public abstract partial class BaseMeshSync : MonoBehaviour, IObservable<MeshSync
 
         #region Properties
         
-        private int currentSessionId = -1;
+        private int  currentSessionId  = -1;
+        private bool ignoreNewSessions = false;
```

Comment:
without more context, ignore new sessions sounds like any new session will be ignored and we will keep the old session, but it seems we are doing the opposite, only for the next session. Maybe a better name is promptOnNextNewSession with inverted boolean values?

Reply:
I've renamed it